packages/backend-common/src/database/DatabaseManager.test.ts

```diff
@@ -340,6 +344,15 @@ describe('DatabaseManager', () => {
       );
     });
 
+    it('returns the same client for the same pluginId', async () => {
+      const client1 = await manager.forPlugin('plugin1').getClient();
+      const client2 = await manager.forPlugin('plugin1').getClient();
```

Comment:
Try this is a `Promise.all` rather individual awaits 💣

Reply:
Ahaha I thought that this might come up actually! :joy: One second, let me see if I can create some failing tests!